app/templates/rentals/show.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="jumbo">
+  <a href="#" class="button right up-11">Edit</a>
```

Comment:
So go ahead and remove edit and we'll add it back when we introduce a forms section